conf/distro/include/rust_security_flags.inc

```diff
@@ -1,4 +1,6 @@
 # Build errors with PIE options enabled
 SECURITY_CFLAGS_pn-rust-native = "${SECURITY_NO_PIE_CFLAGS}"
+SECURITY_CFLAGS_pn-rust-cross-${TARGET_ARCH} = "${SECURITY_NO_PIE_CFLAGS}"
 SECURITY_CFLAGS_pn-rust-cross = "${SECURITY_NO_PIE_CFLAGS}"
```

Comment:
Since we now have `rust-cross-${TARGET_ARCH}` should this line about `rust-cross` be removed?